app/_locales/es/messages.json

```diff
@@ -929,5 +929,579 @@
+  },
+  "customToken": {
+    "message": "Token Personalizado"
+  },
+  "downloadGoogleChrome": {
+    "message": "Descargar Google Chrome"
+  },
+  "dontHaveAHardwareWallet": {
+    "message": "¿No tienes un monedero físico?"
+  },
+  "ensNameNotFound": {
+    "message": "No se localizó el nombre ENS"
+  },
+  "enterPasswordContinue": {
+    "message": "Intoducir contraseña para seguir"
+  },
+  "eth": {
+    "message": "ETH"
+  },
+  "estimatedProcessingTimes": {
+    "message": "Tiempo Previsto de Procesamiento"
+  },
+  "expandView": {
+    "message": "Ampliar Vista"
+  },
+  "fast": {
+    "message": "Rápido"
+  },
+  "faster": {
+    "message": "Más Rápido"
+  },
+  "fastest": {
+    "message": "Lo Mas Rápido"
+  },
+  "feeChartTitle": {
+    "message": "Previsiones en Vivo de las Comisiones por Transacciones"
+  },
+  "forgetDevice": {
+    "message": "Olvidar a este dispositivo"
+  },
+  "functionType": {
+    "message": "Tipo de Función"
+  },
+  "gasUsed": {
+    "message": "Gas Usado"
+  },
+  "gasPriceExtremelyLow": {
+    "message": "Precio de Gas Excesivamente Bajo"
+  },
+  "gasPriceNoDenom": {
+    "message": "Precio de Gas"
+  },
+  "generatingTransaction": {
+    "message": "Generando la Transacción"
+  },
+  "getHelp": {
+    "message": "Pedir Ayuda."
+  },
+  "hardware": {
+    "message": "hardware"
+  },
+  "hardwareWalletConnected": {
+    "message": "Se ha conectado el monedero físico"
+  },
+  "hardwareWallets": {
+    "message": "Conectarle un monedero físico"
+  },
+  "hardwareWalletsMsg": {
+    "message": "Seleccionar un monedero físico que quieres usar con MetaMask"
+  },
+  "havingTroubleConnecting": {
+    "message": "¿Tienes problemas para hacer la conexión?"
+  },
+  "hexData": {
+    "message": "Datos de formato Hex"
+  },
+  "history": {
+    "message": "Historial"
+  },
+  "importUsingSeed": {
+    "message": "Importar usando la frase semilla de la cuenta"
+  },
+  "initialTransactionConfirmed": {
+    "message": "La red confirmó tu transacción inicial. Hazle clic en OK para volver."
+  },
+  "insufficientBalance": {
+    "message": "Saldo insuficiente."
+  },
+  "invalidSeedPhrase": {
+    "message": "Frase semilla no válida."
+  },
+  "keepTrackTokens": {
+    "message": "Estar al tanto con los tokens que has comprado a través de tu cuenta de MetaMask."
+  },
+  "learnMore": {
+    "message": "Más información"
+  },
+  "ledgerAccountRestriction": {
+    "message": "Hay que hacer uso de tu última cuenta antes de agregarle una nueva."
+  },
+  "liveGasPricePredictions": {
+    "message": "Previsiones en Vivo del Precio de Gas"
+  },
+  "menu": {
+    "message": "Menú"
+  },
+  "metamaskSeedWords": {
+    "message": "Palabras semilla de MetaMask"
+  },
+  "metamaskVersion": {
+    "message": "Versión de MetaMask"
+  },
+  "missingYourTokens": {
+    "message": "¿No ves tus tokens?"
+  },
+  "minutesShorthand": {
+    "message": "Min"
+  },
+  "nevermind": {
+    "message": "Olvídatelo"
```

Comment:
I think the translation here is "Olvídalo¨

Reply:
Parece que en algunas partes de latinoamerica, se usa el _te_. Mi argumento es que tanto en Chile, como en los programas de TV latinoamericanos (entiendo que es un mexicano estandarizado), es _Olvídalo_.

Reply:
En España sí se usa el "te". Eso sí, me acostumbro a que mi español peninsular no sea lo corriente internacionalmente. Quito el "te" pos.